### ISSUE TITLE
Question about training data

### ISSUE THREAD

**Issue by Jaswers** [DeepMotionEditing/deep-motion-editing#83]
First of all thank you for sharing such an excellent project, I have a question. When training retargeting, the NPY of the training data contains all the animation data on mixamo (about 2000+) or the author has selected some specified animations. If it is the latter, can you provide a list of these specified animations, or directly provide the bvh files for training, thank you

**Comment by PeizhuoLi** [comment]
The npy files contain all the animation data on Mixamo except those in test set.
